packages/web-components/src/utils/dom.ts

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
I wonder if we should name this file something more descriptive. I'm assuming you named it `dom.ts` because it has to do with sorting dom nodes, but maybe it should be `sort` or something similar. I'd rather have more files with specific names/purposes than fewer files with generic names/purposes.

Reply:
Yeah, I named it that way because of the DOM manipulations and to distinguish it from other less-specific sorting functions. That's not something we need to worry about for now though, so I'll rename it to `dom-sort`.